src/app/pages/StoryPage/index.jsx

```diff
@@ -70,11 +70,22 @@ const StoryPage = ({ pageData, mostReadEndpointOverride }) => {
   const lastPublished = getLastPublished(pageData);
   const aboutTags = getAboutTags(pageData);
   const mostReadInitialData = path(['mostRead'], pageData);
-  const topStoriesInitialData = path(
+  const topStoriesInitialData = pathOr(
+    [],
     ['secondaryColumn', 'topStories'],
     pageData,
   );
-  const featuresInitialData = path(['secondaryColumn', 'features'], pageData);
+  const featuresInitialData = pathOr(
+    [],
+    ['secondaryColumn', 'features'],
+    pageData,
+  );
+
+  const hasTopStoriesData = !!topStoriesInitialData.length;
+  const hasFeaturesData = !!featuresInitialData.length;
+  const removeMostReadLabelTopMargin = !(
+    hasTopStoriesData && hasTopStoriesData
```

Comment:
Is one of these meant to be `hasFeaturesData`?

Reply:
Good spot @HarveyPeachey, Thanks